src/Presentation/Presenters/DonationConfirmationHtmlPresenter.php

```diff
@@ -50,7 +54,8 @@ private function getConfirmationPageArguments( Donation $donation, string $updat
 			],
 			'person' => $this->getPersonArguments( $donation ),
 			'bankData' => $this->getBankDataArguments( $donation->getPaymentMethod() ),
-			'initialFormValues' => $this->getInitialMembershipFormValues( $donation )
+			'initialFormValues' => $this->getInitialMembershipFormValues( $donation ),
+			'piwikTracking' => $piwikTracking->getEvents()
```

Comment:
There is some naming mismatch here. Is it correct to say that `PiwikCustomTracking` collects "piwik events"? Then perhaps it can be renamed to `PiwikEvents`, and `'piwikTracking'` here would become `'piwikEvents'` as well, assuming we can make such a rename.